grafana/dashboards/battery-health.json

```diff
@@ -64,7 +70,6 @@
   "fiscalYearStartMonth": 0,
   "graphTooltip": 1,
   "id": null,
-  "iteration": 1689940704136,
   "links": [

```

Comment:
Is there a specific reason why you want to remove the `iteration` field? I think we should still keep that in place.

Reply:
I'm not aware that I've touched that fact, it's just the export of Grafana itself. Anyway, I've found that importing grafana automatically assigns it.

Reply:
I'm not aware that I've touched that value. It's an export from Grafana itself. I've also found that the dashboard imports without issues. But since it works with that value or not, I'll commit to keep it in place.
Thanks.